lib/integrations/awsoidc/listdatabases.go

```diff
@@ -198,3 +210,22 @@ func listDBClusters(ctx context.Context, clt ListDatabasesClient, req ListDataba
 
 	return ret, nil
 }
+
+func fetchSingleRDSDBInstance(ctx context.Context, clt ListDatabasesClient, req ListDatabasesRequest, clusterID string) (*rdsTypes.DBInstance, error) {
+	describeDBInstanceInput := &rds.DescribeDBInstancesInput{
+		Filters: []rdsTypes.Filter{
+			{Name: &filterDBClusterID, Values: []string{clusterID}},
+		},
+	}
+
+	rdsDBs, err := clt.DescribeDBInstances(ctx, describeDBInstanceInput)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	if len(rdsDBs.DBInstances) == 0 {
+		return nil, trace.BadParameter("DB Cluster %s has no DB Instance", clusterID)
```

Comment:
```suggestion
		return nil, trace.BadParameter("database cluster %s has no DB instances", clusterID)
```

Mostly so we avoid the Capitalized error message?

https://github.com/golang/go/wiki/CodeReviewComments#error-strings